Maddori.Apple/Maddori.Apple/Global/UIComponent/ExitButton.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  ExitButton.swift
+//  Maddori.Apple
+//
+//  Created by 김유나 on 2022/10/18.
+//
+
+import UIKit
+
+import SnapKit
+
+final class ExitButton: UIButton {
+    
+    // MARK: - init
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        configUI()
+    }
+    
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        configUI()
+    }
```

Comment:
p3;

코드의 통일성을 위해 다른 view와 같이 처리하는게 어떨까.. 제안해봅니다.. ^_____^
스토리보드에 관한 init은 실행되지 않아서 그렇습니당..

```suggestion
    required init?(coder: NSCoder) {
        fatalError("init(coder:) has not been implemented")
    }
```

Reply:
수정해서 반영했습니다! 감사합니다 :)